### ISSUE TITLE
执行到此处会卡住，请问是什么问题？The execution will be stuck. What is the problem?

### ISSUE THREAD

**Issue by zhanghaobucunzai** [Randl/MNASNet-pytorch#1]
)
    (18): Sequential(
      (0): Conv2d(320, 1280, kernel_size=(1, 1), stride=(1, 1), bias=False)
      (1): BatchNorm2d(1280, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
      (2): ReLU6(inplace=True)
    )
    (19): AdaptiveAvgPool2d(output_size=1)
  )
  (classifier): Sequential(
    (0): Dropout(p=0.0, inplace=False)
    (1): Linear(in_features=1280, out_features=1000, bias=True)
  )
)
number of parameters: 4383312
FLOPs: 313135872.0
